hamilton/lifecycle/__init__.py

```diff
@@ -1,12 +1,21 @@
-from .api import (
+from .api import (  # noqa: F401
     EdgeConnectionHook,
     GraphAdapter,
     LegacyResultMixin,
     NodeExecutionHook,
     ResultBuilder,
 )
-from .base import LifecycleAdapter
-from .default import PDBDebugger, PrintLnHook
+from .base import LifecycleAdapter  # noqa: F401
+from .default import PDBDebugger, PrintLnHook  # noqa: F401
+
+try:
+    from .conditional_tqdm import TQDMHook  # noqa: F401
+except ImportError:
+    TQDMHook = None
+
+optional = []
```

Comment:
Kinda think this should be default in the driver, no? Only if TQDM is installed?

Reply:
can people remove it? if so, how?